### PR TITLE
Sprint 21-04 initial model changes

### PR DESCRIPTION
**Pull request by florianebmeier**

### CODE REVIEW

File: Log.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import * as isoly from "isoly"
```
Comment (simonmika) [inline Log.ts:1]: Should be from `model-log`.

File: Statistics.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import * as isoly from "isoly"
```
Comment (simonmika) [inline Statistics.ts:1]: Move into `Merchant` folder.

**Review by simonmika — CHANGES_REQUESTED** [review]

File: Log.ts (hunk 1 above)
Comment (florianebmeier) [reply Log.ts:1]: This PR should have been closed, as it was deprecated by old changes

File: Statistics.ts (hunk 2 above)
Comment (florianebmeier) [reply Statistics.ts:1]: Will do so in a new PR

**Pull request closed without merging**